diff --git a/node_modules/priyanshu-fca/src/sendMessage.js b/node_modules/priyanshu-fca/src/sendMessage.js
index 2631d64..abe86d1 100644
--- a/node_modules/priyanshu-fca/src/sendMessage.js
+++ b/node_modules/priyanshu-fca/src/sendMessage.js
@@ -184,7 +184,7 @@ module.exports = function (defaultFuncs, api, ctx) {
       sendContent(form, threadID, false, messageAndOTID, callback);
     } else {
       if (utils.getType(isGroup) != "Boolean")
-        sendContent(form, threadID, threadID.length <= 15, messageAndOTID, callback);
+        sendContent(form, threadID, threadID.length < 15, messageAndOTID, callback);
       else
         sendContent(form, threadID, !isGroup, messageAndOTID, callback);
     }
